lib/templates/json_texture.template.js

```diff
@@ -0,0 +1,43 @@
+path = require("path");
+
+function jsonTextureTemplate(data) {
+  var spriteObj = {};
+
+  // Create frame data for each sprite.
+  spriteObj.frames={};
+
+  data.sprites.forEach(function (sprite) {
+    var entry={
+      frame: {
+        "x": sprite.x,
+        "y": sprite.y,
+        "w": sprite.width,
+        "h": sprite.height
```

Comment:
The linter should enforce that we don't need to quote keys unless they aren't valid identifiers otherwise (e.g. have a `-` or are reserved).

```js
        x: sprite.x,
        y: sprite.y,
        w: sprite.width,
        h: sprite.height
```